src/StorageSync/StorageSync/ServerEndpoint/GetServerEndpointCommand.cs

```diff
@@ -30,6 +31,10 @@ namespace Microsoft.Azure.Commands.StorageSync.Cmdlets
     /// Implements the <see cref="Microsoft.Azure.Commands.StorageSync.Common.StorageSyncClientCmdletBase" />
     /// </summary>
     /// <seealso cref="Microsoft.Azure.Commands.StorageSync.Common.StorageSyncClientCmdletBase" />
+    [CmdletOutputBreakingChange(
```

Comment:
You can remove all of these deprecation messages as you are a preview module and breaking changes are expected.

Reply:
For my learning, how exactly do we know that we are in "preview"?
Also, if breaking changes are OK, should I go ahead and whitelist the ones that static analysis identified?

Reply:
As long as your version is below 1.0.0 you are a preview module: https://github.com/Azure/azure-powershell/blob/master/src/StorageSync/StorageSync/Az.StorageSync.psd1#L15. And yes, feel free to whitelist the breaking changes.

Reply:
Whitelisted the breaking changes:
3010	The property 'CurrentProgress' of type 'Microsoft.Azure.Commands.StorageSync.Models.PSServerEndpointHealth' has been removed.
3030	The generic type for 'property LastSyncPerItemErrorCount' has been changed from 'System.Nullable1[System.Int32]' to 'System.Nullable1[System.Int64]'.